src/ansys/openapi/common/_api_client.py

```diff
@@ -119,7 +119,7 @@ def __call_api(
         body: Optional[Any] = None,
         post_params: Optional[Any] = None,
         files: Optional[Any] = None,
-        response_type: Optional[str] = None,
+        response_type: Union[str, Dict[int, str], None] = None,
```

Comment:
As discussed in https://github.com/pyansys/openapi-client-template/pull/82, I'd prefer having two separate kwargs here. This does mean that the two could conflict, but I think it's just something we'll have to document.